### PR TITLE
/DAMP/VREL - new features

### PR DESCRIPTION
**Pull request by bertrand-altair**
New features for /DAMP/VREL

-> format is changed in 2024 compared to 2023

### CODE REVIEW

File: engine/source/assembly/damping_vref_rby.F90 [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+!Copyright>        You should have received a copy of the GNU Affero General Public License
+!Copyright>        along with this program.  If not, see <https://www.gnu.org/licenses/>.
+!Copyright>
+!Copyright>
+!Copyright>        Commercial Alternative: Altair Radioss Software
+!Copyright>
+!Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+!Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+!Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+! ======================================================================================================================
```
Comment (laurent-altr) [inline engine/source/assembly/damping_vref_rby.F90:23]: Each new file should contain a module that embed the subroutines

**Review by laurent-altr — CHANGES_REQUESTED** [review]

**Review by sebastienVilleneuve — APPROVED** [review]

File: engine/source/assembly/damping_vref_rby.F90 [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+! ----------------------------------------------------------------------------------------------------------------------
+          implicit none
+! ----------------------------------------------------------------------------------------------------------------------
+!                                                   Included files
+! ----------------------------------------------------------------------------------------------------------------------
+#include "my_real.inc"       
+! ----------------------------------------------------------------------------------------------------------------------
+!                                                   Arguments
+! ----------------------------------------------------------------------------------------------------------------------
+          type(GROUP_),                              intent(in) :: igrnod(ngrnod)
```
[the remark this answers is not in the document]
Comment (bertrand-altair) [reply engine/source/assembly/damping_vref_rby.F90]: done

File: engine/source/assembly/damping_vref_rby.F90 [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+!Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+!Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+! ======================================================================================================================
+!                                                   PROCEDURES
+! ======================================================================================================================
+!
+!=======================================================================================================================
+!                           Computation of damping forces for /DAMP/VREL with RBODY
+!=======================================================================================================================
+!
```
[the remark this answers is not in the document]
Comment (bertrand-altair) [reply engine/source/assembly/damping_vref_rby.F90:32]: done

File: engine/source/assembly/damping_vref_rby.F90 (hunk 1 above)
Comment (bertrand-altair) [reply engine/source/assembly/damping_vref_rby.F90:25]: done

**Review by sebastienVilleneuve — APPROVED** [review]

**Review by laurent-altr — APPROVED** [review]

**Pull request merged**